dotnet-petclinic-payment/PetClinic.PaymentService/readme.md

```diff
@@ -0,0 +1,7 @@
+# Local Dev Guide
+
+The PetClinic uses Eureka Services discovery, to test this application locally you need to run Eureka Server, use the following command to run it using docker
+
+```
+ docker run -d --publish 8761:8761 public.ecr.aws/o9d5t0i3/eureka-server:latest
```

Comment:
I don't think it's reasonable to use a repo from non-verified account like this `public.ecr.aws/o9d5t0i3/eureka-server`. Can we give instruction for customers to build the image themselves?

Reply:
removed as it's not needed.